EcommerceApp/frontend/angular-ecommerce/README.md

```diff
@@ -38,3 +39,30 @@ This project was generated with [Angular CLI](https://github.com/angular/angular
 
 Run `ng serve` for a dev server. Navigate to `http://localhost:4200/`. The app will automatically reload if you change any of the source files.
 
+=======
```

Comment:
It looks like git conflict, you should resolve it properly

Reply:
Done